marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -1084,3 +1084,29 @@ describe('RTL Source Selector/Source Search tests', () => {
     });
 });
 
+describe('Source URI navigation', () => {
+
+    afterEach(cleanup);
+
+    test('Index resets on close of mapping',  async() => {
+        const { getByLabelText, getByTestId } = render(<SourceToEntityMap {...data.mapProps}  mappingVisible={true}/>);
+
+        // URI index starts at 1
+        let uriStart = within(getByLabelText('uriIndex'));
+        expect(uriStart.getByText('1')).toBeInTheDocument();
+
+        // Click next, URI index is 2
+        fireEvent.click(getByTestId('navigate-uris-right'));
+        let uriNext = within(getByLabelText('uriIndex'));
+        expect(uriNext.getByText('2')).toBeInTheDocument();
+
+        // Close mapping modal
+        fireEvent.click(getByLabelText('icon: close'));
+
+        // URI index reset to 1
+        let uriReset = within(getByLabelText('uriIndex'));
+        expect(uriReset.getByText('1')).toBeInTheDocument();
+    });
```

Comment:
This is a good test. Couple of nitpicky items:
1. Do we need a separate test suite? We tried to test URI navigation and verify source table data, didnt work out at the time. If we are able to assert doc content on navigation, then we can keep this in a separate suite and add more tests later to assert source data.
2. uriStart, uriNext, uriReset are really aliases to the same locator?